plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/LinuxExec.kt

```diff
@@ -40,11 +44,14 @@ open class LinuxExec @Inject constructor(objects: ObjectFactory) : ConclaveTask(
                     conclaveBuildDir
                 )
             } catch (e: Exception) {
+                logger.info("Docker build of conclave-build failed. Re-run your build with '--info' and " +
+                        "submit a new GitHub issue https://github.com/R3Conclave/conclave-core-sdk/issues/new", e)
```

Comment:
I don't think we need to mention submitting a GH issue here. It could be they find out what the issue is by enabling --info and fix the issue themselves. Mentioning it in the exception message is sufficient.

Reply:
Hopefully this now make sense.